oh_staff_ui/views_utils.py

```diff
@@ -117,28 +117,73 @@ def get_result_items(queryset_list: list) -> list[ProjectItem]:
     return output_projectitems_deduped
 
 
-def get_search_results(search_type: str, query: str) -> list[ProjectItem]:
+def get_search_results(
+    search_type: str,
+    query: str,
+    item_type_filter: str,
+    media_file_type_filter: str,
+    status_filter: str,
+) -> list[ProjectItem]:
     # Return a list of items matching the search.  Different searches have
     # different return types and sort orders; this unifies them for consistent
     # use in the view and template.
-    if search_type == "title":
+
+    # first, check if this is a wildcard search - if so, return all items
+    # no need to check search_type
+    if query == "*":
+        qs_results = ProjectItem.objects.all().order_by("title")
+        results_list = [item for item in qs_results]
+        filtered_results = filter_results_list(
```

Comment:
No need for this identical line in all 4 cases.  Since it does the same thing for each case, just move it below them, right above the `return` statement.